src/components/SearchBar/SearchBar.tsx

```diff
@@ -2,7 +2,7 @@ import { useState } from 'react';
 import './SearchBar.css';
 import Button from '../Button/Button';
 
-function SearchBar({ onSearch }) {
+function SearchBar({ onSearch }: any) {
```

Comment:
I think if you make your function a type, you should be able to export it and set it here rather than using any.

Reply:
Have you looked in your browser console?

I have now made a proper type for the prop. Thanks

Reply:
Yes, sorry - I resolved that comment because I looked in my browser! And thanks :)